README.md

```diff
@@ -75,15 +75,6 @@ You can speed this library up ~100x by using C-bindings to [bitcoin core's `libs
 
 ### `libsecp256k1` Dependency Installation
 
-
-#### Easy (MacOS only)
-Use [brew](https://brew.sh/) to install `libsecp256k1`:
```

Comment:
Why get rid of this?

Reply:
That used to be true, but I fixed that upstream in 2021!
https://github.com/cuber/homebrew-libsecp256k1/commit/50639e19866e11bcfe3c2b3860ce6b73cd12f733

Reply:
ah, then the upstream libsecp256k1 needs to upgrade to the latest version because that one still uses the deprecated function. How do I do that?

Reply:
It looks like you just change the version number and hash digest here:
https://github.com/mflaxman/homebrew-libsecp256k1/blob/master/libsecp256k1.rb

(You can do it on your own branch/repo and use `$ brew tap` for testing)